CONTRIBUTING.md

```diff
@@ -17,50 +23,96 @@ and contributions of all forms are welcome!
 
 You can contribute by:
 
-- Submitting bug reports and features requests on the GitHub
-  [issue tracker](https://github.com/slundberg/shap/issues), or
-- Contributing fixes and improvements via
-  [Pull Requests](https://github.com/slundberg/shap/pulls).
+- Submitting bug reports and features requests on the GitHub [issue
+  tracker][issues],
+- Contributing fixes and improvements via [Pull Requests][pulls], or
+- Discussing ideas and questions in the [Discussions forum][discussions].
 
-## Installing from the master branch
+If you are looking for a good place to get started, look for issues with the
+[good first issue][goodfirstissue] label.
+
+[issues]: https://github.com/shap/shap/issues
+[pulls]: https://github.com/shap/shap/pulls
+[discussions]: https://github.com/shap/shap/discussions
+[goodfirstissue]:
+    https://github.com/shap/shap/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
+
+## Writing helpful bug reports
+
+When submitting bug reports on the [issue tracker][issues], it is very helpful
+for the maintainers to include a good **Minimal Reproducible Example** (MRE).
+
+An MRE should be:
+
+- **Minimal**: Use as little code as possible that still produces the same
+  problem
+- **Self-contained**: Include everything needed to reproduce your problem,
+  including imports and input data.
+- **Reproducible**: Test the code you're about to provide to make sure it
+  reproduces the problem
```

Comment:
Let's be consistent here with ending the sentences in these lists with a period `.`.